js/instructions.js

```diff
@@ -0,0 +1,15 @@
+const targetOpen = document.querySelector('.open')
+const targetClose = document.querySelector('.close')
+const targetInstructions = document.querySelector('.rules-text')
```

Comment:
I think these variable names could be better. They make sense in the context of this file, but you may end up with more code here, in which case these names are not specific enough.

Make the names more semantic and obvious

Reply:
Done